result.php

```diff
@@ -0,0 +1,184 @@
+<?php
+require 'vendor/autoload.php';
+use Dotenv\Dotenv;
+$dotenv = Dotenv::createMutable(__DIR__);
+$dotenv->load();
+$apiKey = $_ENV["apiKey"];
+
+// Declaring array for storing error messages in array format.
+$errorArray = [];
+$fullName = "";
+$trimmed = "";
+// Storing regex for checking.
+$namePattern = '/^[a-zA-Z]+$/';
```

Comment:
`$namePattern = '/^[a-zA-Z]+$/';`
Explain more about this regular expression. What is it checking?